sdk/ai/azure-ai-projects/samples/evaluations/sample_agent_response_evaluation.py

```diff
@@ -0,0 +1,138 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to create and run an evaluation for an Azure AI agent response
+    using the synchronous AIProjectClient.
+
+    The OpenAI compatible Evals calls in this sample are made using
+    the OpenAI client from the `openai` package. See https://platform.openai.com/docs/api-reference
+    for more information.
+
+USAGE:
+    python sample_agent_response_evaluation.py
+
+    Before running the sample:
+
+    pip install "azure-ai-projects>=2.0.0" azure-identity python-dotenv
+
+    Set these environment variables with your own values:
+    1) AZURE_AI_PROJECT_ENDPOINT - The Azure AI Project endpoint, as found in the Overview
+       page of your Azure AI Foundry portal.
+    2) AZURE_AI_MODEL_DEPLOYMENT_NAME - The deployment name of the AI model, as found under the "Name" column in
+       the "Models + endpoints" tab in your Azure AI Foundry project.
+"""
+
+import os
+import time
+from pprint import pprint
+from dotenv import load_dotenv
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects import AIProjectClient
+from azure.ai.projects.models import (
+    PromptAgentDefinition
+)
+
+load_dotenv()
+
+project_client = AIProjectClient(
+    endpoint=os.environ["AZURE_AI_PROJECT_ENDPOINT"],
+    credential=DefaultAzureCredential(),
+)
+
+with project_client:
+
+    # [START agent_response_evaluation]
+    openai_client = project_client.get_openai_client()
```

Comment:
The #[START... and #[END...  comments are only needed if you plan to embed code snippets in the package README.md file. I would remove them from now, and until you (or someone else) decide what to show in the package readme.